lib/mayaUsd/fileio/primUpdaterManager.h

```diff
@@ -103,14 +153,64 @@ class PrimUpdaterManager : public PXR_NS::TfWeakBase
     //! Create the pull parent and set it into the prim updater context.
     MDagPath setupPullParent(const Ufe::Path& pulledPath, VtDictionary& args);
 
+    //! Record pull information for the pulled path, for inspection on
+    //! scene changes.
+    void recordPullVariantInfo(const Ufe::Path& pulledPath, const MDagPath& pullParentPath);
+
+    // Maya file new or open callback.  Member function to access other private
+    // member functions.
+    static void beforeNewOrOpenCallback(void* clientData);
```

Comment:
Flush the variant configuration information in the Trie on file new or open.

Reply:
I think I'd rather have the function be name on what processing it does than what triggers it, but this is minor.